src/vision/harmonic.md

```diff
@@ -0,0 +1,31 @@
+# 🌈 Harmonic synthesis
+
+This page is a **work in progress**. It represents a complete vision for where we want async to go. This vision starts with the [experience] that we want async to provide and then dives into details of the technical pieces that need to get done to create that vision. Note that while a lot of the steps needed are fairly clear, several of them also have [significant unknowns or points of controversy](./harmonic/unresolved_questions.md). We have attempted to highlight those and expect to be working through those points as we go.
```

Comment:
I think this could be separated into a **What is this?** part like the rest to be clearer.